disnake/guild.py

```diff
@@ -3752,6 +3752,36 @@ def widget_image_url(self, style: WidgetStyle = WidgetStyle.shield) -> str:
         """
         return self._state.http.widget_image_url(self.id, style=str(style))
 
+    async def edit_mfa_level(self, mfa_level: MFALevel, *, reason: Optional[str] = None) -> None:
+        """|coro|
+
+        Edits the two-factor authentication level of the guild.
+
+        You must be the guild owner to use this.
+
+        .. versionadded:: 2.6
+
+        Parameters
+        ----------
+        mfa_level: :class:`int`
+            The new 2FA level. If set to 0, the guild does not require
+            2FA for their administrative members to take
+            moderation actions. If set to 1, it does.
+        reason: Optional[:class:`str`]
+            The reason for editing the mfa level. Shows up on the audit log.
+
+        Raises
+        ------
+        HTTPException
+            Editing the 2FA level failed.
+        ValueError
+            You are not the owner of the guild.
+        """
+        if self.owner_id != self._state.self_id:
+            raise ValueError("To edit the 2FA level, you must be the owner of the guild.")
+        # return value unused
+        await self._state.http.edit_mfa_level(self.id, mfa_level, reason=reason)
```

Comment:
we should catch the typecheck issue of mfa_level being a bool

Reply:
63507c4b437f775218216b09738764d55e2c9472